### PR TITLE
feat: Add status-go version to About screen

### LINKED ISSUE #11424: Add status-go version to About screen
### Description

So QAs and users can tell which version is used in case of incompatibilities. @anastasiyaig asked for it.

### Acceptance Criteria

- status-go version (commit version works but tag is better as it's what mobile uses) is shown in the about page

### PR DESCRIPTION
**Pull request by caybro**
- Settings/About now displays the `status-go` version, read from its `VERSION` file
- use the actual app icon, to differentiate between prod/dev version
- make the version numbers clickable, taking the user to either the release notes (prod) or the GH commit browser (dev)
- add "Is production" switch to the About page in storybook

Closes #11424

### Affected areas

Settings/About

### Screenshot of functionality (including design for comparison)

- [x] I've checked the design and this PR matches it

image

Storybook, prod:
image

Storybook, dev:
image

### CODE REVIEW

File: src/constants.nim [hunk 1]
```diff
@@ -81,10 +81,11 @@ proc ensureDirectories*(dataDir, tmpDir, logDir: string) =
 
 # This is changed during compilation by reading the VERSION file
 const DESKTOP_VERSION {.strdefine.} = "0.0.0"
+const STATUSGO_VERSION* {.strdefine.} = "0.0.0"
```
Comment (jrainville) [inline src/constants.nim:84]: It seems to work, but I don't get how. I tried searching for `STATUSGO_VERSION` and I can't see it. Did you forget to push a Makefile change?

Comment (caybro) [reply src/constants.nim:84]: Ouch :D of course!

Comment (caybro) [reply src/constants.nim:84]: Then I wonder _HOW_ is this working for you...

Comment (jrainville) [reply src/constants.nim:84]: > Then I wonder _HOW_ is this working for you...

I have not tested. I just meant that it seems to work in your screenshots

Comment (caybro) [reply src/constants.nim:84]: Right; I pushed the missing Makefile changes

**Review by igor-sirotin — APPROVED** [review]
LGTM 👍

**Review by anastasiyaig — APPROVED** [review]

**Pull request merged**